src/addressBook/index.js

```diff
@@ -309,7 +307,7 @@ export async function getIndexDatasetFromAddressBook(
 ) {
   const { respond, error } = createResponder();
   try {
-    const addressBookIri = `${getSourceUrl(addressBookDataset)}#this`; // TODO: Ugly hack, should remove
+    const addressBookIri = `${getSourceIri(addressBookDataset)}#this`; // TODO: Ugly hack, should remove
```

Comment:
AFAICT `getSourceIri` is just an alias for `getSourceUrl`, so I would prefer that we stay consistent in our use of `getSourceUrl`.